src/main/java/Main.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Main entrypoint to the program.
+ */
+public class Main {
+
+    public static void main(String[] args) {
+        UI ui = UI.getUiInstance();
+        Unker unker = Unker.getUnkerInstance();
```

Comment:
Can consider adding a comment to explain what "Unker" is as it may not be obvious what it is to another reader.